doc/src/build/programming-with-objects/ch1-object-basics.md

```diff
@@ -105,7 +105,8 @@ First of all, we begin the test with a hardcoded test address, which will also g
 ```rust
 let owner = @0x1;
 // Create a ColorObject and transfer it to @owner.
-let scenario = &mut test_scenario::begin(&owner);
```

Comment:
:facepalm: I didn't spot these docs needed changing, thanks for catching.  Should we also include these changes in the update to the docs site for today? cc @randall-Mysten

Reply:
I can push another update once this PR is merged